src/changes.rs

```diff
@@ -148,6 +150,26 @@ impl<'a> ChangeSet<'a> {
                       -> Result<Option<String>, ::std::io::Error> {
         let text = &self.file_map[filename];
 
+        fn style<T>(mut writer: T,
```

Comment:
This function could do with a better name, or a comment, its not clear what it does, to me.